### PR TITLE
Fix kflash.py invocation, basic board detection

### PR DESCRIPTION
**Pull request by dkm**
Remove -v from kflash.py invocation (was displaying version info and exiting)
Add a list of boards to test and exits if none could be found.

### CODE REVIEW

File: k210-run [hunk 1]
```diff
@@ -27,4 +27,27 @@ FW="$(dirname "$ELF")/firmware.bin"
 riscv64-unknown-elf-objcopy -O binary "$ELF" "$FW"
 
 dir=$(dirname $(readlink -f $0))
-"$dir/kflash.py" --port /dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0 $OPTS --terminal -v "$FW"
+
+declare -a known_serial_ids
+known_serial_ids[0]=/dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0
```
Comment (Disasm) [inline k210-run:32]:
```suggestion
# Sipeed MAIX Dock board
known_serial_ids[0]=/dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0
```

Comment (Disasm) [reply k210-run:32]: Oops, can't apply the change myself from the GH interface, will push it later

**Review by Disasm — APPROVED** [review]
